tests/php_test_files/issue_185.php

```
@@ -0,0 +1,15 @@
+<?php
+
+use Spiral\Goridge;
+require "vendor/autoload.php";
```

PHPStan (2.2.5)

[error] Path in require() "vendor/autoload.php" is not a file or it does not exist.


(require.fileNotFound)